src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Common/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SFA.DAS.ApprenticeAan.Application.Common
+{
+    public static class Constants
+    {
+        public static class AanHubApiUrls
+        {
+            private const string ApiPrefix = "/api";
+            public const string GetRegionsUrl = $"{ApiPrefix}/regions";
```

Comment:
Don't see a point of having specific urls as constants. This pattern will become complicated when you have dynamic routes to build.

Reply:
IMHO it is just unnecessary hop in the code to check the URL. I am expecting that one request will have one endpoint to associate with, so dont think there is any benefit. If we find a benefit of doing this down the line then we can always refactor.